views/layouts/main.handlebars

```diff
@@ -2,14 +2,37 @@
 <html>
 
 <head>
-  <meta charset="utf-8">
-  <title>{{title}}</title>
+ <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>NODOMON</title>
```

Comment:
Change to Donomon
I mistakenly said nodomon before

Reply:
done